terraform/alerting/module.service/main.tf

```diff
@@ -0,0 +1,5 @@
+resource "google_monitoring_custom_service" "service" {
+  service_id   = var.service_name
+  display_name = var.service_name
+  project      = var.project
+}
```

Comment:
Can you configure your editor to add a newline at EOF? Terraform will accept it, but it'll cause lots of unnecessary future diffs on PRs.

Reply:
Updated VS Code setting for this, thank you.

Reply:
You can also ask vscode to fmt for you too 👍